### ISSUE TITLE
[Question] Very bold font.

### ISSUE THREAD

**Issue by roman-tiukh** [grafana/grafana-zabbix#570]
### I'm submitting a ...
- [ ] Bug report
- [ ] Feature requestdd
- [X] Question / Support request

![bold](https://user-images.githubusercontent.com/38906004/39802396-957fda34-5376-11e8-81a8-91d2ae1cb532.PNG)

Привет, почему на панели триггеров установлен такой жирный шрифт? Глаза режет, буквы слипаются? Как его изменить?

**Comment by heyrod** [comment]
I experience the same issue. The triggers panel is hardly readable, especially on big resolutions. That came out with the last update of Grafana and Zabbix plugin - I don't know which of them to be blamed.

**Comment by alexanderzobnin** [comment]
Yes, seems, original `font-weight: 500` is overwritten by plugin's CSS (set it to `bold` which looks too bold). So which one is better, what do you think?
1. Current (`font-weight: bold`)
2. Grafana original (`font-weight: 500`)
3. Not bold (`font-weight: 300`)

<img width="428" alt="screen shot 2018-06-01 at 12 12 10" src="https://user-images.githubusercontent.com/4932851/40832993-4e861216-6595-11e8-9102-2538b742e81b.png">
<img width="440" alt="screen shot 2018-06-01 at 12 14 49" src="https://user-images.githubusercontent.com/4932851/40833057-81320076-6595-11e8-869d-82fb888a3afc.png">
<img width="422" alt="screen shot 2018-06-01 at 12 15 19" src="https://user-images.githubusercontent.com/4932851/40833067-84c8a30c-6595-11e8-8e22-e1d5f699dfe5.png">

**Comment by frogour** [comment]
3. Not bold.
Красиво и читабельно.
Nice and readable.
